docs/source/rules.rst

```diff
@@ -42,15 +42,31 @@ then it should flag on the segment FOLLOWING, the place that the desired
 element is missing.
 
 
-In-line Ignoring Errors
+Inline Ignoring Errors
 -----------------------
-`sqlfluff` features in-line error ignoring. For example, the following will
-ignore the lack of whitespace surounding the ``*`` operator.
+`sqlfluff` features inline error ignoring. For example, the following will
+ignore the lack of whitespace surrounding the ``*`` operator.
 
 .. code-block:: sql
 
-   a.a*a.b AS bad_3  -- noqa: L006
+   a.a*a.b AS bad_1  -- noqa: L006
 
 Multiple rules can be ignored by placing them in a comma-delimited list.
+
+.. code-block:: sql
+
+   a.a *  a.b AS bad_2,  -- noqa: L007, L006
```

Comment:
Does this work even with the space? I thought only `-- noqa: L007,L006` worked but I may be wrong

Reply:
[The space should not cause an issue.](https://github.com/sqlfluff/sqlfluff/blob/master/src/sqlfluff/core/linter.py#L739)
>                 rules = [r.strip() for r in comment_remainder.split(",")]